crates/torii/graphql/src/tests/models_test.rs

```diff
@@ -46,54 +51,78 @@ mod tests {
 
         let value = run_graphql_query(&pool, query).await;
 
-        let moves_mdoels = value.get("movesModels").ok_or("no moves found").unwrap();
+        let moves_models = value.get("movesModels").ok_or("no moves found").unwrap();
         let moves_connection: Connection<Moves> =
-            serde_json::from_value(moves_mdoels.clone()).unwrap();
+            serde_json::from_value(moves_models.clone()).unwrap();
 
-        let position_mdoels = value.get("positionModels").ok_or("no position found").unwrap();
+        let position_models = value.get("positionModels").ok_or("no position found").unwrap();
         let position_connection: Connection<Position> =
-            serde_json::from_value(position_mdoels.clone()).unwrap();
+            serde_json::from_value(position_models.clone()).unwrap();
 
         assert_eq!(moves_connection.edges[0].node.remaining, 10);
-        assert_eq!(position_connection.edges[0].node.x, 42);
-        assert_eq!(position_connection.edges[0].node.y, 69);
+        assert_eq!(position_connection.edges[0].node.vec.x, 42);
+        assert_eq!(position_connection.edges[0].node.vec.y, 69);
     }
 
-    #[ignore]
+    //#[ignore]
     #[sqlx::test(migrations = "../migrations")]
     async fn test_model_where_filter(pool: SqlitePool) {
-        entity_fixtures(&pool).await;
+        let mut db = Sql::new(pool.clone(), FieldElement::ZERO).await.unwrap();
+
+        entity_fixtures(&mut db).await;
 
         // fixtures inserts two position mdoels with members (x: 42, y: 69) and (x: 69, y: 42)
         // the following filters and expected total results can be simply calculated
         let where_filters = Vec::from([
-            ("where: { x: 42 }", 1),
-            ("where: { xNEQ: 42 }", 1),
-            ("where: { xGT: 42 }", 1),
-            ("where: { xGTE: 42 }", 2),
-            ("where: { xLT: 42 }", 0),
-            ("where: { xLTE: 42 }", 1),
-            ("where: { x: 1337, yGTE: 1234 }", 0),
-            (r#"where: { player: "0x2" }"#, 1), // player is a key
+            (
+                r#"where: { playerNEQ: "0x0000000000000000000000000000000000000000000000000000000000000002" }"#,
+                1,
+            ),
+            (
+                r#"where: { playerGT: "0x0000000000000000000000000000000000000000000000000000000000000002" }"#,
```

Comment:
Ah we should trim zeros on input objects too, much like graphql returned results. We can do that in another PR